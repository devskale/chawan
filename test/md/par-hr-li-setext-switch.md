test

hi
***
hi

hi
===
hi

hi
---
hi

* test
---
* test
  
  test
---

===
test

test
* test
test
