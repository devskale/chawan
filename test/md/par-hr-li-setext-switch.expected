test

hi

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

hi

[1m[4m#[24m hi[m

hi

[1m[4m##[24m hi[m

hi

  • test

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  • test

    test

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

=== test

test

  • test test
